apps/docs/content/docs/usage.mdx

```diff
@@ -25,28 +23,28 @@ export default function TldrawExample() {
 
 ### CSS
 
-In order to use the `<Tldraw/>` component, you should also import the `tldraw.css` file from the `@tldraw/tldraw` library. You can alternatively import this file inside of another CSS file using the `@import` syntax.
+In order to use the `<Tldraw>` component, you should also import the `tldraw.css` file from the `@tldraw/tldraw` package. You can alternatively import this file inside of another CSS file using the `@import` syntax.
```

Comment:
Changed library to package to match some recent changes that I've done elsewhere. I think there is **one** tldraw library, which contains multiple packages. It's one very good product, with multiple parts. Not lots of little products.

Also, throughout this file, I'm changing components from 
`<Tldraw/>` to `<Tldraw>`

I think it's more readable, and it's consistent with the Introduction page.
I do think it's good to write in as a component though! It reminds people that it's just a React component.